src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/WorkspaceProjectSnapshotChangeTrigger.cs

```diff
@@ -82,12 +103,12 @@ internal void Workspace_WorkspaceChanged(object sender, WorkspaceChangeEventArgs
                         // is saved, or loses focus in the editor.
                         project = e.OldSolution.GetProject(e.ProjectId);
                         var document = project.GetDocument(e.DocumentId);
-                        
+
                         // Using EndsWith because Path.GetExtension will ignore everything before .cs
                         // Using Ordinal because the SDK generates these filenames.
                         if (document.FilePath != null && document.FilePath.EndsWith(".cshtml.g.cs", StringComparison.Ordinal))
                         {
-                            EnqueueUpdate(e.ProjectId);
+                            EnqueueWithDelay(e.ProjectId);
```

Comment:
Why change stuff like this?

Reply:
lol oops, ya i'll undo. Moved code around a bit and had more stuff in this class at one poitn.